docs/source/user-guide/compatibility.md

```diff
@@ -116,6 +116,7 @@ The following cast operations are generally compatible with Spark except for the
 | string | long |  |
 | string | binary |  |
 | string | date | Only supports years between 262143 BC and 262142 AD |
+| string | timestamp |  |
```

Comment:
To mark this cast as compatible, we would need to enable the `cast StringType to TimestampType` fuzz test in CometCastSuite, and this currently fails due to incompatibilities between Spark and Comet.

Reply:
Got it, I mistook fuzz tests as running fuzz-test jar through SPARK-SUBMIT. Will revert the compatibility-guide changes.